### ISSUE TITLE
Publish to Organization's repository.

### ISSUE THREAD

**Issue by DevotionGeo** [benweet/stackedit#1231]
The **publish to github** using **Jekyll Site** template is a great option, but it doesn't work correctly when you want to publish to a repo inside a github organization. 
For example I can publish to some repository created under my github profile, but can't publish to the repository (elixirbook.github.io) under the organization (elixirbook) I created.

Thank You!

**Comment by jshleap** [comment]
Any advances on this? I have tried to publish in an organization I created and keeps not synchronizing.

**Comment by queglay** [comment]
I can't publish anything to an organisation, and my user is allowed to push to that organisation.
I can publish to any repo in my own user account, but as soon as I try an organisation I get a 404 error.

**Comment by dgiovannelli** [comment]
Hi all, any follow up on this? This could be a major dealbreaker for me...

**Comment by stevenjswanson** [comment]
Me too.  My team wants to use stackedit specifically for this use case.  It's awesome, but we need org. support.

Thanks for building this!

**Comment by Fred-White94** [comment]
I agree with the above, shame it does not allow syncing with an organisation page (even for the owner)
